.github/workflows/update-schemas.yaml

```diff
@@ -0,0 +1,72 @@
+name: Update schemas in repository
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 3 * * *'
+
+jobs:
+  update-store:
+    strategy:
+      matrix:
+        include:
+          - environment: infra
+            cluster_url: https://api.moc-infra.massopen.cloud:6443
+            cluster_secret: INFRA_SA_TOKEN
+          - environment: smaug
+            cluster_url: https://api.smaug.na.operate-first.cloud:6443
+            cluster_secret: SMAUG_SA_TOKEN
+          - environment: morty
+            cluster_url: https://api.morty.emea.operate-first.cloud:6443
+            cluster_secret: MORTY_SA_TOKEN
+          - environment: rick
+            cluster_url: https://api.rick.emea.operate-first.cloud:6443
+            cluster_secret: RICK_SA_TOKEN
+          - environment: balrog
+            cluster_url: https://api.balrog.aws.operate-first.cloud:6443
+            cluster_secret: BALROG_SA_TOKEN
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get schemas from cluster
+        uses: ./.github/actions/build-schemas
+        with:
+          OPENSHIFT_SERVER: ${{ matrix.cluster_url }}
+          OPENSHIFT_TOKEN: ${{ secrets[matrix.cluster_secret] }}
+        continue-on-error: true
+
+  prepare-pr:
+    runs-on: ubuntu-22.04
+    needs: [update-store]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Move to workspace folder
+        shell: bash
+        run: |
+          cd $GITHUB_WORKSPACE
+
+      - name: Download all workflow artifacts
+        uses: actions/download-artifact@v3
+
+      - name: Move skip_kinds.txt to root folder
+        shell: bash
+        run: |
+          cd schemas
+          mv skip_kinds.txt ../
+          cd schemas
+          cp -r * ../
+          cd ..
+          rm -rf schemas
```

Comment:
I'm confused, isn't this the same as:

```suggestion
          mv schemas/* .
          rm -rf schemas
```

Reply:
This was interesting because when I had it setup like this it threw error which is on the image below
![image](https://user-images.githubusercontent.com/47832967/181560518-169662ab-cc12-427b-82c8-c2c5468e9c05.png)

From what I have found `mv` will refuse to move folders which contains different files and folders and this kind of bypasses that

Reply:
Resolved this by using `cp -r` and had to change name of artifacts because otherwise they would confuse `rm` command